pkg/agent/openflow/pipeline.go

```diff
@@ -1157,6 +1383,54 @@ func (f *featurePodConnectivity) flowsToTrace(dataplaneTag uint8,
 	return flows
 }
 
+// flowsToTrace is used to generate flows for Traceflow in featureService.
+func (f *featureService) flowsToSample(dataplaneTag uint8,
+	ovsMetersAreSupported,
+	senderOnly bool,
+	receiverOnly bool,
+	packet *binding.Packet,
+	endpointPackets []binding.Packet,
+	ofPort uint32,
+	timeout uint16) []binding.Flow {
+	cookieID := f.cookieAllocator.Request(cookie.PacketSampling).Raw()
+	var flows []binding.Flow
+
+	// Clear the loaded DSCP bits before output.
+	ifLiveTraffic := func(fb binding.FlowBuilder) binding.FlowBuilder {
```

Comment:
Why you use `ifLiveTraffic` in this function, but use `clearDSCP` in featurePodConnectivity.flowsToSample for the same purpose? It is better to define a unique function, and call it in the different features?

Reply:
updated.